python/hsfs/__init__.py

```diff
@@ -59,6 +57,3 @@ def get_sdk_info():
 
 
 __all__ = ["connection", "disable_usage_logging", "get_sdk_info"]
-# running async code in jupyter throws "RuntimeError: This event loop is already running"
-# with tornado 6. This fixes the issue without downgrade to tornado==4.5.3
-nest_asyncio.apply()
```

Comment:
After digging deeper,  I realised some of the problems were caused by `nest_asyncio` as it makes the actual async errors difficult to debug. So I tried to remove its dependency, but its still needed in case of jupyter, otherwise the user has to explicitly call `nest_asyncio.apply()` in each notebook. It is not needed for python runtimes. So it is needed for below cases:
1.  jupyter ->  added check in init_async methods
2. locust -> added into the `locustfile`